### DISCUSSION TITLE
How to change + Rule to "Create your query"

### DISCUSSION THREAD

**Discussion by lytechxdigital** [react-querybuilder/react-querybuilder#753] [Q&A] [answered]
Hello Team i an new in  query builder i try but not Sucess i want to change + rule name to other so can you write me a code so i can use that for change my name . 
![Screenshot (1)](https://github.com/user-attachments/assets/af46c8f1-761a-4dbd-a44a-77c46d783bac)

**Comment by jakeboone02** [comment, marked as answer]
Use the [`translations` prop](https://react-querybuilder.js.org/docs/components/querybuilder#translations) like this:

```tsx
 return <QueryBuilder translations={{ addRule: { label: "Create your query" } }} />
```

**Reply by lytechxdigital** [reply]
Thank you so much ...

**Reply by jakeboone02** [reply]
No problem. I don't normally pitch my training course here on the repo but I have a feeling you'd benefit from it: https://www.newline.co/courses/building-advanced-admin-reporting-in-react

**Reply by lytechxdigital** [reply]
i just add a new query can you help me : #754

**Reply by jakeboone02** [reply]
Thanks, I keep track of all new issues and discussions.
